lib/pluginmanager/update.rb

```diff
@@ -6,6 +6,7 @@ class LogStash::PluginManager::Update < LogStash::PluginManager::Command
   REJECTED_OPTIONS = [:path, :git, :github]
 
   parameter "[PLUGIN] ...", "Plugin name(s) to upgrade to latest version", :attribute_name => :plugins_arg
+  option "--[no-]verify", :flag, "verify plugin validity before installation", :default => true
```

Comment:
Well, my intent with this PR was to fix the plugin manager for private repos, I tried to put a descriptive title:

```
Fix plugin_manager install and update commands to work properly with no internet env private gem repos
```

I apologies for the confusions.